lib/graphql/metrics/tracer.rb

```diff
@@ -90,6 +93,18 @@ def capture_multiplex_start_time
         yield
       end
 
+      def capture_lexing_time
+        # GraphQL::Query#result fires `lex` before the `execute_multiplex` event, so sometimes
```

Comment:
This lack of reliability might have also been part of the reason we veered away from depending on `lex` at one point.

Reply:
I'm actually confused by this, I copied it from the parse method but I can't figure out when it would _not_ be fired before `execute_multiplex` except when it's not fired at all?

I don't know if there's changes across gem versions or something, but there's a lot of comments and ordering notes in here that don't make a lot of sense to me.